app/views/broker_agencies/quotes/_upload_employee_modal.html.erb

```diff
@@ -3,7 +3,7 @@
   <h2><%= l10n("upload_employee_roster")%></h2>
   <h4><%= l10n("broker_agencies.quotes.template_upload")%></h4>
   <% file_name = EnrollRegistry[:census_employee_roster].settings(:roster_upload_file_name).item %>
-  <%= link_to(document_download_path(env_bucket_name("templates"), s3_template_bucket_key) + "?contenttype=application/vnd.ms-excel&filename=#{file_name}", class: "download") do %>
+  <%= sanitize link_to(document_employees_template_download_path + "?contenttype=application/vnd.ms-excel&filename=#{file_name}", class: "download") do %>
```

Comment:
Can you please double check this link **Need the template?**. It looks like broken to me. I am seeing the link url instead of the text.